### PR TITLE
Commonize CSS interfaces and remove polyfill dependency

### PR DESCRIPTION
**Pull request by Skolotsky**
CSS interfaces now have implementation in K/JVM (with TODO() fallback for now), in K/JS and in native Browser structures.
`useNativeCSSTypedOM` is used for switching API which used for style applying.
Removed polyfill dependency

### CODE REVIEW

File: web/core/src/commonMain/kotlin/androidx/compose/web/css/CSSTypedOM.kt [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+@file:Suppress("UNUSED")
+package androidx.compose.web.css
+
+import kotlin.js.JsName
+
+expect interface CSSStyleValue
+
+open class CSSStyleValueKT : CSSStyleValue {
+    companion object {
+        fun parse(property: String, cssText: String): CSSStyleValue = TODO()
```
Comment (eymar) [inline web/core/src/commonMain/kotlin/androidx/compose/web/css/CSSTypedOM.kt:15]: is TODO() left intentionally?

Comment (Skolotsky) [reply web/core/src/commonMain/kotlin/androidx/compose/web/css/CSSTypedOM.kt:15]: yes, it isn't used yet, and i think that maybe it should be removed from interfaces now

File: web/core/build.gradle.kts [hunk 2]
```diff
@@ -30,7 +30,6 @@ kotlin {
         val jsMain by getting {
             dependencies {
                 implementation(kotlin("stdlib-js"))
-                implementation(npm("css-typed-om", "0.4.0"))
```
Comment (Schahen) [inline web/core/build.gradle.kts:33]: Can you please remove css-typed-om license from licenses as well?

File: web/core/src/commonMain/kotlin/androidx/compose/web/css/CSSKT.kt [hunk 3]
```diff
@@ -0,0 +1,107 @@
+/*
```
Comment (Schahen) [inline web/core/src/commonMain/kotlin/androidx/compose/web/css/CSSKT.kt:1]: Not a big deal but it seems to be that this file can have better (= more intuituve) name )

File: web/core/src/commonMain/kotlin/androidx/compose/web/css/CSSTypedOM.kt [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+    companion object {
+        operator fun invoke(value: kotlin.String): String
+
+        operator fun invoke(value: kotlin.Number): Number
+
+        operator fun invoke(value: CSSStyleValue): StyleValue
+    }
+}
+
+// type CSSNumberish = number | CSSNumericValue
```
Comment (Schahen) [inline web/core/src/commonMain/kotlin/androidx/compose/web/css/CSSTypedOM.kt:44]: I suggest to name it `CSSNumeric`  - it's honestly equally understandable and a bit more simple English.

**Review by Schahen — COMMENTED** [review]
Apart from three comments I left the only thing I would like you to change is package names - 
let's keep all behind `org.compos.jetbarains.web` prefix

File: web/core/src/commonMain/kotlin/androidx/compose/web/css/CSSTypedOM.kt (hunk 4 above)
Comment (Skolotsky) [reply web/core/src/commonMain/kotlin/androidx/compose/web/css/CSSTypedOM.kt:44]: I just reused name from TypeScript d.ts typing for it. But CSSNumeric is better, agree)

File: web/core/src/commonMain/kotlin/androidx/compose/web/css/CSSKT.kt (hunk 3 above)
Comment (Skolotsky) [reply web/core/src/commonMain/kotlin/androidx/compose/web/css/CSSKT.kt:1]: That object in  Browser API named just `CSS`, and it is kotlin implementation of that object... So i have another name CSSKotlinImpl.kt

File: web/core/build.gradle.kts (hunk 2 above)
Comment (Skolotsky) [reply web/core/build.gradle.kts:33]: Sure!

File: web/core/src/commonMain/kotlin/androidx/compose/web/css/CSSKT.kt (hunk 3 above)
Comment (Skolotsky) [reply web/core/src/commonMain/kotlin/androidx/compose/web/css/CSSKT.kt:1]: Do you have better naming for that?

Comment (Schahen) [reply web/core/src/commonMain/kotlin/androidx/compose/web/css/CSSKT.kt:1]: I usually take the "biggest" entity in the file and name the file accordingly but, again, it's no a big deal )

**Pull request merged**